lib/fabric/app/models/fabric/invoice.rb

```diff
@@ -92,35 +93,33 @@ def sync_with(invoice)
       self.collection_method = invoice.collection_method
       self.created = invoice.created
       self.currency = invoice.currency
-      self.customer_id = invoice.customer
+      self.customer_id = handle_expanded(invoice.customer)
       self.custom_fields = invoice.custom_fields&.map do |e|
-        e.to_hash.with_indifferent_access
+        handle_hash(e.to_hash)
```

Comment:
this should just be passed `e`, not `e.to_hash`, right?

Reply:
Yep, forgot to remove that extra `.to_hash`.